--- ./node_modules/wrtc/lib/peerconnection.js	2016-08-14 21:52:57.000000000 +0200
+++ ./node_modules/wrtc/lib/peerconnection.js	2017-02-01 18:52:27.691387962 +0100
@@ -1,9 +1,4 @@
-var path = require('path');
-var binary = require('node-pre-gyp');
-var path = require('path');
-//var binding_opts = require(path.resolve(path.join(__dirname, '../build/wrtc.json')));
-var binding_path = binary.find(path.resolve(path.join(__dirname,'../package.json')));
-var _webrtc = require(binding_path);
+var _webrtc = eval('require')('./wrtc.node');
 
 var EventTarget = require('./eventtarget');
 
